### PR TITLE
Update national/international hotline phone numbers (closes #12)

### LINKED ISSUE #12: Outdated phone numbers in repository details "About" description
## Which hotline

App and TAN

## What is missing

The phone numbers +498007540001/2 in the "About" information in this [repository](https://github.com/corona-warn-app/cwa-hotline) are outdated. The 800 number is only available from within Germany.

image

## Why should it be included

The information should be updated to correspond to the phone numbers in use today.

## Suggested change

Change 

"+498007540001/2"

in "About" to 

"08007540001/2 (national) and +493049875401/2 (international)" 

to correspond to hotline phone numbers published on https://www.coronawarn.app/en/. (Click on gear-wheel to the right of "About" to access "Edit repository details".)

### PR DESCRIPTION
**Pull request by MikeMcC399**
This PR updates the [README.md](https://github.com/corona-warn-app/cwa-hotline/blob/master/README.md) with the current hotline phone numbers as shown on https://www.coronawarn.app/en/ . It resolves the issue #12.

image

This information has been added to [README.md](https://github.com/corona-warn-app/cwa-hotline/blob/master/README.md) in the form of a table:

Hotline | App | TAN
--- | --- | ---
National | [0800 7540001](tel:08007540001) | [0800 7540002](tel:08007540002)
International | [+49 30 498 75401](tel:+493049875401) | [+49 30 498 75402](tel:+493049875402).

The new numbers were introduced in March 2021 - see https://github.com/corona-warn-app/cwa-documentation/issues/502#issuecomment-800893642.

---
Additional incidental changes:
- I updated the copyright to 2020-2021
- I removed some trailing spaces at the end of lines

### CODE REVIEW

**Comment by MikeMcC399** [comment]
@dsarkar 
I forgot to link the issue and PR, so I corrected that now. Please merge the PR if you agree with the changes. I believe they are correct.

**Comment by MikeMcC399** [comment]
Waiting for approval and merge by Open Source Team.

**Comment by Ein-Tim** [comment]
@heinezen It seems like you're on shift today, so maybe you'd like to take a look at this? (mentioning because @heinezen is not watching this repository) 🙂

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks!

**Comment by dsarkar** [comment]
Sorry for late merge .... got somehow lost ...

**Pull request merged**
